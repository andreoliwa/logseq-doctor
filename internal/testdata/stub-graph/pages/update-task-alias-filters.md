filters:: {"onedrive" false, "tax" false, "some other page" true, "gnucash" true}
alias:: other name for page

- TODO My task key
