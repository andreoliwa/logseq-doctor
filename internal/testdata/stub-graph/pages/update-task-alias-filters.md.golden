filters:: {"onedrive" false, "tax" false, "some other page" true, "gnucash" true}
alias:: other name for page

- TODO My updated task with a key
